src/store/ngoSignup.js

```diff
@@ -0,0 +1,54 @@
+import update from "immutability-helper";
+import { connect } from "react-redux";
+import { applyScope } from "./utils";
+import { pageSelector } from "./selectors";
+
+const scope = "ngoSignup";
+
+const initialState = {
+  mode: "ngosignup",
```

Comment:
unless there's another mode, like variations of NGO form the mode related code can be removed.